cluster-autoscaler/cloudprovider/packet/packet_manager_rest.go

```diff
@@ -318,9 +320,15 @@ func createEquinixMetalManagerRest(configReader io.Reader, discoverOpts cloudpro
 		klog.Fatalf("No \"default\" or [Global] nodepool definition was found")
 	}
 
-	metalAuthToken := os.Getenv("PACKET_AUTH_TOKEN")
-	if len(metalAuthToken) == 0 {
-		klog.Fatalf("PACKET_AUTH_TOKEN is required and missing")
+	var metalAuthToken string
+	value, present := os.LookupEnv(metalAuthTokenEnv)
+	if present {
+		metalAuthToken = value
+	} else {
+		metalAuthToken = os.Getenv("PACKET_AUTH_TOKEN")
+		if len(metalAuthToken) == 0 {
+			klog.Fatalf("%s or PACKET_AUTH_TOKEN is required and missing", metalAuthTokenEnv)
```

Comment:
in this fallback perhaps we should say `%s or METAL_AUTH_TOKEN is required and missing` instead of the legacy `PACKET_AUTH_TOKEN`

Reply:
Its the same thing `METAL_AUTH_TOKEN or PACKET_AUTH_TOKEN  is required and missing` as I am passing `metalAuthToken`

Reply:
might be helpful to match the pattern of using a constant for PACET_AUTH_TOKEN.